src/Haskell/Ide/Engine/Transport/LspStdio.hs

```diff
@@ -216,18 +224,22 @@ mapFileFromVfs :: (MonadIO m, MonadReader REnv m)
 mapFileFromVfs tn vtdi = do
   let uri = vtdi ^. J.uri
       ver = fromMaybe 0 (vtdi ^. J.version)
+  lf <- asks lspFuncs
   vfsFunc <- asksLspFuncs Core.getVirtualFileFunc
   mvf <- liftIO $ vfsFunc (J.toNormalizedUri uri)
   case (mvf, uriToFilePath uri) of
-    (Just (VFS.VirtualFile _ yitext _), Just fp) -> do
-      let text' = Rope.toString yitext
+    (Just (VFS.VirtualFile _ _ _), Just _fp) -> do
```

Comment:
Not sure what's happening here. It looks like I just didn't implement it at first.

Reply:
> 
> 
> Not sure what's happening here. It looks like I just didn't implement it at first.

Yeah, I dont really know either the purpose of that.

Reply:
We should talk about this tomorrow. It looks wrong in some way.